var/spack/repos/builtin/packages/elbencho/package.py

```diff
@@ -27,6 +28,7 @@ class Elbencho(MakefilePackage):
     version("2.1-1", sha256="18be49f521df2fab4f16a1a9f00dd6104a25e5ea335ce8801bf07268ed9271a9")
     version("2.0-9", sha256="fe0f67fbb7dd7c743f8b3e0a92358f7393f2950da456474d4adb38690fab1878")
     version("2.0-7", sha256="a2e49cb2cf1ae99e46e9fa95b42ece250cb58fbadb4c393f9776b40204e8b2c0")
+    version("master", branch="master")
```

Comment:
By convention, infinite versions such as master, develop, or main should typically be placed above numbered versions. (Don't worry Spack will continue to use numbered versions by default unless otherwise specified.)